docker/readme.md

```diff
@@ -1,13 +0,0 @@
-# Utiliser Road2 avec Docker 
-
```

Comment:
You can keep this file. It is for the description of the directory, not for the static website.

Reply:
Ok, you moved it but github didn't see it...

Reply:
Yes I moved it but I don't understand why it's marked as deleted. Maybe because I changed some of the content.